c2/cuda/groth16_cuda.cu

```diff
@@ -6,6 +6,10 @@
 #include <mutex>
 #include <algorithm>
 
+#define SPPARK_DONT_INSTANTIATE_TEMPLATES
+
+//#define G2_MSM_ON_GPU
```

Comment:
To summarize. The actual change is merely a choice to split msms based on population counts. So I'd say that it's appropriate to remove the traces of the experiment, and on later point revisit the option to "steal" the jobs.